src/utils/nostr_utils.py

```diff
@@ -18,22 +18,19 @@
 NOSTR_RELAYS = ["wss://nostr-pub.wellorder.net", "wss://relay.damus.io"]
 NOSTR_WATCH = "https://api.nostr.watch/v1/online"
 
-def shuffle_relays():
-    if (input(f"Default relays: {NOSTR_RELAYS}\nShuffle more relays? y/n ")) == 'y':
-        r = (requests.get(NOSTR_WATCH, {})).json()
-        while(1):
-            new_relays = [x for x in r[:15] if 'damus' not in x and 'nostr-pub.wellorder' not in x]
-
-            if (ans := input(f"New online relays found:\n{new_relays}\nIs that OK? y/n/r (n to go back to original, r to reshuffle) ")) == 'y':
-                for val in new_relays: 
-                    NOSTR_RELAYS.append(val)
-                break
-            elif ans == 'r':
-                random.shuffle(r)
-            elif ans == 'n':
-                break
-
-def add_relays():
+def show_relays():
+    r = (requests.get(NOSTR_WATCH, {})).json()
+    new_relays = [x for x in r[:20] if 'damus' not in x and 'nostr-pub.wellorder' not in x]
```

Comment:
Is that just to skip protocol portion of the url?

Reply:
- First question: that’s because I’ve let the original relays in the NOSTR_RELAY global list, so I’m excluding them here to avoid potential duplicates
- Second question: sorry for the magic number, should be corrected, that’s an arbitrary number bc the list of active relays is quite long, originally I was asking the user if he wanted to reshuffle so that it shows 20 other relays and so on

The relay selection could be improved based on relay location or other informations

Reply:
Would this make the service dependent on Nostr Watch? If that service goes down would that be a single point of failure? We may be able to achieve the same thing with an async websocket relay pool unless there are other advantages to using Nostr Watch.

Reply:
Yeah the show option relies on the nostr_watch API , the advantage to me is that it's dynamically updated which could be desirable given how relays are prone to instability.
Moreover it seems pretty [canonical](https://github.com/fiatjaf/nostr-relay-registry).
If by websocket relay pool you mean a statically maintained pool, it could lead to extra continuous work to maintain if but if you know how to get relays that are online and dynamically update the pool, it would indeed be a better idea. I could peep the nostr_watch repo to see how they get the relays under the hood.
Lmk what you think